templates/user.mustache

```diff
@@ -26,6 +26,14 @@
         "userpicture": ""
     }
 }}
-<div id="mod_kanban_assignee-{{cardid}}-{{id}}" class="mod_kanban_assigned_user btn btn-icon icon-no-margin p-0" data-id="{{cardid}}" data-userid="{{id}}" title="{{fullname}}">
+<div id="mod_kanban_assignee-{{cardid}}-{{id}}" class="mod_kanban_assigned_user btn btn-icon icon-no-margin p-0" data-id="{{cardid}}" data-userid="{{id}}" title="{{fullname}}" data-toggle="tooltip" data-placement="top">
     {{{userpicture}}}
-</div>
\ No newline at end of file
+</div>
+{{#js}}
+    document.getElementById('mod_kanban_assignee-{{cardid}}-{{id}}').addEventListener('click', function(e) {
+        $('#mod_kanban_assignee-{{cardid}}-{{id}}').tooltip('show');
+        setTimeout(function() {
+            $('#mod_kanban_assignee-{{cardid}}-{{id}}').tooltip('hide');
+        }, 2000);
+    });
+{{/js}}
```

Comment:
new line at end of file please.

Could you please put the JS code into a separate module? JS code here really should be as minimalistic as possible (which basically is "load module") due to coding guidelines AFAIK

Reply:
As the Moodle tooltip function relies on jQuery, I have to use it here too. Alternative would be to build an own tooltip function.